### DISCUSSION TITLE
Need to scale vault-configurer pods to 3

### DISCUSSION THREAD

**Discussion by kapil7148** [bank-vaults/bank-vaults#2309] [Q&A] [answered]
Hi Team ,

I am using vault-operator helm chart , by deploying helm chart of vault-operator , i am getting pods for vault, vault-operator and vault-configurer , and also able to scale up the pods for vault and vault-operator but not able to see how to increase replicas for vault-configurer pods and also from where deployment is created for vault-configurer

Can someone help me to increase the replicas for vault-configurer using code or crds where i need to add replicas code .

Thanks in advance

**Comment by akijakya** [comment]
Hi @kapil7148, there is currently no way of my knowing to increase the replicas of the configurer via configuration, only manually by editing the already existing deployment. Although I don't really see the reason for it, I think for what it is doing, 1 pod should be enough!

**Comment by akijakya** [comment, marked as answer]
As this was also asked [here](https://github.com/bank-vaults/vault-operator/discussions/255), I am now closing this discussion.
